### PR TITLE
re-formatted sub-schema

### PR DESCRIPTION
**Pull request by vmaineng**

### CODE REVIEW

**Review by hjkmines — APPROVED** [review]

File: server/models/User.js [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
+      type: mongoose.Schema.Types.ObjectID,
+      ref: "User",
+    },
+  }, {
+    timestamps: true
+  });
+
 const UserSchema = new Schema(
   {
     userName: {
```
Comment (FunkyColdMedyna) [inline server/models/User.js:28]: should this have a `type:String` as well ? It looks like the userName object has the profileImage object inside of it

Comment (hjkmines) [reply server/models/User.js:28]: @FunkyColdMedyna can you elaborate more what you mean by `userName` having the `profileImage` object? I show `profileImage` as its own separate entity inside the `UserSchema`

Comment (FunkyColdMedyna) [reply server/models/User.js:28]: //what. i see on the diff //
....
{
   userName: {
       profileImage:{
       type: String,
   },
ratings : [RatingSchema]
}, ....

should this be...

...
{
   userName: {
      type: String,
      },
   profileImage: {
     type: String,
     },
    ratings: [RatingSchema],
    }, ....

**Comment by JosephThawn** [comment]
Looks great to me!

**Review by JosephThawn — APPROVED** [review]

**Review by FunkyColdMedyna — APPROVED** [review]

**Pull request merged**